.github/workflows/pull_request.yml

```diff
@@ -41,9 +28,30 @@ jobs:
           reporter: github-pr-review
           path: '.'
           pattern: '*.sh'
+          
+      - uses: cachix/install-nix-action@v22
+      - name: Check if the chart follows a valid semver version
+        run: |
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' scripts/nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+          branch_name="${{ github.event.pull_request.base.ref }}"
+          nix-shell --pure --run "./scripts/validate-version.sh --branch $branch_name" ./scripts/shell.nix
 
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.6.1
```

Comment:
nit: Conflicting dependency sourcing. Should we source our dependencies only from one of the sources (i.e. nix-shell or github actions plugins)?

Reply:
Well, ideally yes, but using gh-actions at some places gives us some more than just the binary or the tool.

Reply:
I changed it to happen via nix-shell.